libflux/core/src/semantic/tests.rs

```diff
@@ -79,6 +81,113 @@ impl Importer for HashMap<&str, PolyType> {
     }
 }
 
+fn is_monotype (
+    ty: types::Record,
+    ret: Option<MonoType>
+) -> Option<MonoType> {
+    match ty {
+        types::Record::Empty => {ret},
+        types::Record::Extension{head: h, tail: t} => {
+        match ret {
+            Some(mono) => {
+                if mono == h.v {
+                    match t {
+                        MonoType::Record(rect) => is_monotype(*rect, Some(h.v)),
+                        _ => None,
+                    }
+                } else {
+                    None
+                }
+            },
+            None => {
+                match t {
+                    MonoType::Record(rect) => is_monotype(*rect, Some(h.v)),
+                    _ => None,
+                }
+			},
+		}
+    }
+}
+}
+
+fn convert_record_constraints (
```

Comment:
What does this function do?

Reply:
That function is just a helper function that converts record constraints that are just monotypes back to their corresponding monotype so that the tests pass.